index.js

```diff
@@ -55,29 +55,17 @@ module.exports = fp(
     function (
         fastify,
         {
-            dsn,
-            environment = 'Local',
+            environment,
             release,
-            autoSessionTracking = false,
-            defaultIntegrations = false,
-            integrations,
             allowedStatusCodes = defaultAllowedStatusCodes,
             onErrorFactory = defaultErrorFactory,
             ...sentryOptions
         },
         next
     ) {
-        Sentry.init({
-            dsn,
-            environment,
-            release,
-            autoSessionTracking,
-            defaultIntegrations,
-            integrations,
-            ...sentryOptions,
-        });
-
-        if (!dsn) {
+        sentryOptions = sentryOptions || {};
+        Sentry.init(sentryOptions);
```

Comment:
Why now don't you pass as input the environment variables to the sentry Client?

Reply:
I am not sure if I understand what you mean, but the environment variables are parsed by the Sentry SDK

Reply:
What I mean is that now you have unpacked the environment, and inside `sentryOptions` is no more present.

Reply:
Ok, the `environment` is still setup to a default value by the sdk and I think it might be the same value we used. The other variables are either the ones we should not override anymore, or didn't have default value.

Reply:
Ok, thanks for the clarification.